.github/CODEOWNERS

```
@@ -122,6 +122,11 @@ tslint.config.js @felixfbecker
 /cmd/frontend/internal/pkg/usagestats @dadlerj
 /pkg/eventlogger @dadlerj
 
+# a8n
+*/a8n/*
+/enterprise/pkg/a8n
+/pkg/a8n
```

```suggestion
/pkg/a8n
**/campaigns/**
```

This includes the webapp files and also cmd/frontend/graphqlbackend/campaigns.go